README.md

```diff
@@ -199,15 +199,15 @@ Please follow the steps mentioned at [here](https://docs.microsoft.com/en-us/azu
 3. There are two ways to run LISAv2 tests:
   * Note when testing with 'Azure' platform:
 
-    * '-ARMImageName' is optional when testing with Azure Gallery Images ('-OsVHD' is not provided to Run-LISAv2.ps1 parameters). If '-ARMImageName' and '-OsVHD' are not provided from Run-LISAv2.ps1 parameters, LISAv2 will try to use <DefaultARMImageName> from .\XML\GlobalConfigurations.xml as value of '-ARMImageName'. In this case, if <DefaultARMImageName> is not defined from .\XML\GlobalConfigurations.xml, exeption will be thrown.
+    * `-ARMImageName` is optional when testing with Azure Gallery Images (`-OsVHD` is used for testing with custom image). If both `-ARMImageName` and `-OsVHD` are not provided from Run-LISAv2.ps1 parameters, LISAv2 will try to use `<DefaultARMImageName>` from .\XML\GlobalConfigurations.xml as value of `-ARMImageName`. In this case, if `<DefaultARMImageName>` is not defined from `.\XML\GlobalConfigurations.xml`, exeption will be thrown.
```

Comment:
why those changes shown in your PR?

Reply:
That was rebase problem. I will remove it.

Reply:
reverted and gone.